releasenotes/notes/preset-pm-vf2-max-trials-958bb8a36fff472f.yaml

```diff
@@ -0,0 +1,18 @@
+---
+upgrade:
+  - |
+    The maximum number of trials evaluated when searching for the best
+    layout using :class:`.VF2Layout` is now limited in
+    :func:`qiskit.transpiler.preset_passmanagers.level_1_pass_manager`,
+    :func:`qiskit.transpiler.preset_passmanagers.level_2_pass_manager`,
+    and
+    :func:`qiskit.transpiler.preset_passmanagers.level_3_pass_manager`
+    to ``2,500``, ``25,000``, and ``250,000``, respectively. Previously,
+    all possible layouts were evaluated. To perform a full search as
+    before, manually run :class:`.VF2PostLayout` over the transpiled circuit,
+    in strict mode, specifying ``0`` for ``max_trials``.
```

Comment:
This needs a little tweaking both to explain the rationale behind why we're making this change, and also elaborating on exactly how to restore the previous behavior with an example. But, I'll just do this manually in the release notes roundup as we prepare 0.24.0

Reply:
Makes sense to add an example, though does the rationale belong in the upgrade text section?

Reply:
Well for things like this where we're making a potentially breaking change I always stress the importance of the "why" so we're explaining to users why we're potentially breaking them. It just provides context to users so hopefully they understand the rationale behind why we're changing the behavior of the code and aren't as angry with us :) . In this case we have a good justification because if we didn't truncate the search we would be spending a near infinite amount of time evaluating different layouts.

Reply:
I added a bit more context in 9c0e5e8. Perhaps we should still add a code example in the release notes roundup as you mentioned.